src/Sensors.cpp

```diff
@@ -19,14 +19,20 @@ void Sensors::loop() {
             if(_onDataCb) _onDataCb();
             dataReady = true;            // only if the main sensor is ready
         }else{
-            if(_onErrorCb)_onErrorCb("-->[W][SENSORS] PM sensor not configured!");
```

Comment:
why you remove this line? or why put in the next if?

Reply:
undo to the original